front-end/src/Components/Reports/reports.resource.tsx

```diff
@@ -0,0 +1,101 @@
+const url = process.env.REACT_APP_URL;
+export const getCounties = async () => {
+  return await fetch(url + `/county`)
+    .then((response) => {
+      return response.json();
+    })
+    .then((res: any) => {
+      return res.data;
+    })
+    .catch((error: any) => {
+      return error;
+    });
+};
```

Comment:
Since we are repeating this line of code, we could write a fetcher function that encapsulate this logic, such that we only give it the endpoint and expect data from the function, This should be done in another PR later, at the moment this okay
something like this
```
const useFetch = (url: string, init?: RequestInit) => {
  return window
    .fetch(url, init)
    .then((response) => {
      return { data: response.json(), ...response };
    })
    .catch((error) => error);
};
```